dxapp_compliance/dxapp_queries.py

```
@@ -906,17 +908,135 @@ def get_latest_commit_date(self, organisation_name, repo_name, token):
+                return False
+
+            # List contents of repo (root directory)
+            contents_url = f"https://api.github.com/repos/{self.ORGANISATION}/{repo_name}/contents"
+            contents_response = requests.get(contents_url, headers=headers)
+            contents_response.raise_for_status()
+
+            contents = contents_response.json()
+
+            # Search for 'requirements.txt' without case-sensitivity
+            file_exists = any(
+                item['type'] == 'file' and 'requirements.txt' == item['name'].lower()
```

Ruff (0.12.2)

Yoda condition detected

Rewrite as `item['name'].lower() == 'requirements.txt'`

(SIM300)